log/slog/slog.go

```diff
@@ -21,28 +24,31 @@ const defaultErrKey = "error"
 var (
 	defaultLoggerWrapper *loggerWrapper
 	logLevel             string
+	logSensitive         bool
 	callerDepth          int
 )
 
 func init() {
 	logLevel = env.String("LOG_LEVEL", "info")
+	logSensitive = env.Bool("LOG_SENSITIVE", true)
```

Comment:
Why does this have to be an env?

Reply:
for default sensitivity. can hardcode as false